src/provider/model/Model.ts

```diff
@@ -208,6 +215,13 @@ export class Model {
         }
     }
 
+    private async onApiHandlerConnection(identity: Identity): Promise<void> {
+        if (await this._environment.getEntityType(identity) === EntityType.EXTERNAL_CONNECTION) {
```

Comment:
is a .net application guaranteed to be EntityType.EXTERNAL_CONNECTION? what happens to the iframes inside the .net apps?

Reply:
Raised this with #dc-adapters people - yeah, seems that OpenFin iframes and .NET embedded views will both be identified as `"iframe"`.

Will need to think about the implications of this - we may even need a test .NET app before we can solve properly. Will add to the second part of this story, or log separately in the backlog.

Reply:
alright